### DISCUSSION TITLE
running Nuxt3 in SSR mode but with static assets served from CDN

### DISCUSSION THREAD

**Discussion by drewda** [nuxt/nuxt#26769] [Questions] [answered]
We are running a Nuxt 3 web app in SSR mode (in a Docker container on our Kubernetes cluster). I'm considering how we can improve overall page-load performance by serving some of the static assets like images, CSS, and the non-dynamic JavaScript from a CDN.

Are there any recommended patterns for how to upload just the static assets from a Nuxt 3 build to a CDN and to change the base URL for just those assets in the compiled HTML?

Thank you!

**Comment by xjccc** [comment, marked as answer]
You can use config.
[cdnURL](https://nuxt.com/docs/api/nuxt-config#cdnurl)

**Reply by drewda** [reply]
Thanks for the pointer! I should have noticed that in the documentation.

Is there also a way to change where the `_nuxt` directory is served from?

**Reply by xjccc** [reply]
You can look at this [build.assetsDir](https://nuxt.com/docs/api/nuxt-config#build-1)

**Reply by drewda** [reply]
O.K., thanks again!
